tests/unit_tests/cpu/sf_finder_brute_force.cpp

```diff
@@ -95,13 +95,13 @@ GTEST_TEST(detray_surface_finders, brute_force_search) {
 
     using detector_t = decltype(det);
 
-    // dummy track
-    detail::ray<typename detector_t::transform3> trk({0.f, 0.f, 0.f}, 0.f,
-                                                     {0.f, 0.f, 1.f}, -1.f);
-
     // Now run a brute force surface search in the first barrel layer
     dindex test_vol_idx{7UL};
     const auto& vol = det.volume_by_index(test_vol_idx);
 
+    // track in x-direction
```

Comment:
Volume 7 is a barrel layer, rather use a track in x direction to find some surfaces